cycle/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3.5"
```

Comment:
Why you changed the version of the docker-compose file?

Reply:
That was a mistake. I changed it back.